src/index.js

```diff
@@ -1,11 +1,11 @@
 // eslint-disable-next-line
```

Comment:
- [ ] You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is robust code and clean.

Reply:
it was disabled just one line to the lodash library, but anyway, this module is not used and deleted the line and the module